index.js

```diff
@@ -204,7 +210,11 @@ module.exports = {
         pull.asyncMap((msg, cb) => {
           // TODO: call ssb-db2 reindexEncrypted
 
-          if (lodashGet(msg, 'value.content.recps', []).includes(ssb.id)) {
+          if (
+            lodashGet(msg, 'value.content.recps', []).includes(
+              fromFeedSigil(ssb.id)
```

Comment:
probably don't need to calculate this every time, could extract variable.
Also this is gonna change with meta-feeds - this needs to be the root id. (ssb.id is the `main` leaf id)

Reply:
i think andre is changing this in the metafeeds pr